vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/FieldSteps.java

```diff
@@ -104,6 +120,46 @@ public WebElement clearFieldLocatedByUsingKeyboard(Locator locator)
         return element;
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b> and checks that it is
+     * sorted by text with using <b>sortingOrder</b> by element text.
+     * List should have more than 1 elements. If some elements don't
+     * contain text it will be displayed in the report.
+     * @param locator locator to locate elements
+     * @param sortingOrder sorting order<br>
+     * <i>Available rule:</i>
+     * <ul>
+     * <li><b>ASCENDING</b> - will check that the list is sorted alphabetically,
+     * <li><b>DESCENDING</b> - will check that the list is sorted reverse alphabetically
+     * </ul>
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void areElementSorted(Locator locator, SortingOrder sortingOrder)
```

Comment:
I think the step should be moved to `WebElementsSteps` since it does not work with fields